tests/tensorflow2/test_keras_mirrored.py

```diff
@@ -260,6 +262,10 @@ def test_save_all(out_dir, tf_eager_mode):
         )
         # weights, grads, optimizer_variables, metrics, losses, outputs
     assert len(tr.steps()) == 3
+    for tname in tr.tensor_names():
+        assert len(tr.tensor(tname).workers(0)) == (
+            1 if workers == "one" else strategy.num_replicas_in_sync
+        )
```

Comment:
now that you made this change,  can you merge the tests test_save_one_worker and test_save_all_workers here combining all the relevant assert statements?

Reply:
done